packages/button/stories/button.stories.tsx

```diff
@@ -42,13 +44,21 @@ Basic.args = {
 
 export const AllColorScheme = () => {
   return (
-    <div style={{ display: 'flex', flexWrap: 'wrap', gap: '0.5rem' }}>
-      <Button colorScheme="default" className="sda sada">
-        Button
-      </Button>
-      <Button colorScheme="primary">Button</Button>
-      <Button colorScheme="secondary">Button</Button>
-      <Button colorScheme="danger">Button</Button>
-    </div>
+    <YoruProvider
+      config={{ initialColorMode: 'light', useSystemColorMode: true }}
+      theme={{
+        colors: {
+          primary: 'hotpink',
+        },
+      }}
+    >
+      <div style={{ display: 'flex', flexWrap: 'wrap', gap: '0.5rem' }}>
+        <Button variant="primary" className="sdsda">
```

Comment:
Since you already pull to my branch `feat/themes-config` and If you are already done with this button story please return it to its original condition, because we don't use primary, or secondary, etc variants anymore. And this is should be an error linter why husky still passing this one. maybe I'll check it later.